### PR TITLE
Google Drive and Amazon S3 Data Source: changed to force character data type when merged as a single data frame.

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

As for Google Drive and Amazon S3 Data Source, changed to force character data type when merged as a single data frame.

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -2441,6 +2442,14 @@ read_excel_file <- function(path, sheet = 1, col_names = TRUE, col_types = NULL,
   if(!is.null(tzone)) { # if timezone is specified, apply the timezeon to POSIXct columns
     df <- df %>% dplyr::mutate_if(lubridate::is.POSIXct, funs(lubridate::force_tz(., tzone=tzone)))
   }
+  # When this API is called from getExcelFilesFromS3, getExcelFilesFromGoogleDrive, and read_excel_files,
+  # by default the convertDataTypeToChar is set as TRUE to covert the resulting data frame columns' data type as character.
+  # This is required to make sure that merging the Excel based data frames doesn't error out due to column data types mismatch.
+  # Once the data frames merging is done, readr::type_convert is called from Exploratory Desktop to restore the column data types.
+  # We don't want to reply on readxl::read_excel's col_types argument "text" since this option converts Date or POSIXct column data as number instead of "2021-01-01" style text.
```
Comment (hideaki) [inline R/system.R:2449]: Typo: reply => rely

File: R/system.R [hunk 2]
```diff
@@ -2441,6 +2442,14 @@ read_excel_file <- function(path, sheet = 1, col_names = TRUE, col_types = NULL,
...
+  if(convertDataTypeToChar) {
```
Comment (hideaki) [inline R/system.R:2450]: Adjust style.

File: R/system.R (hunk 1 above)
Comment (hidekoji) [reply R/system.R]: Fixed typo.

File: R/system.R (hunk 2 above)
Comment (hidekoji) [reply R/system.R]: fixed style.

File: R/system.R (hunk 1 above)
Comment (hidekoji) [reply R/system.R]: fixed typo.

File: R/system.R (hunk 2 above)
Comment (hidekoji) [reply R/system.R]: fixed style.

**Pull request merged**